dotnet/src/Microsoft.Agents.AI.Workflows/FanOutEdgeData.cs

```diff
@@ -37,6 +38,11 @@ internal FanOutEdgeData(string sourceId, List<string> sinkIds, EdgeId edgeId, As
     /// </summary>
     public AssignerF? EdgeAssigner { get; }
 
+    /// <summary>
+    /// An optional label for the edge.
+    /// </summary>
+    public string? Label { get; }
```

Comment:
[Minor] This should probably be on EdgeData, rather than the specific EdgeDatas.